README.md

```diff
@@ -763,155 +763,124 @@ You can use the `ilab model evaluate` command to evaluate the models you are tra
 | MTBenchBranch | Skills | N/A | Tests your skill contributions against a judge model and produces a score based on the difference in performance | N/A |
 
 > [!NOTE]
+> Evaluation must be used with local models (safetensors or GGUF format). Using models directly from Hugging Face without downloading them is unsupported.
 > MTBench and MTBenchBranch use [prometheus-8x7b-v2.0](https://huggingface.co/prometheus-eval/prometheus-8x7b-v2.0) as the judge model by
 default. While you do not need to use this model as your judge, it is strongly recommended to do so if you have the necessary hardware
 resources. You can download it via `ilab model download`.
 
 #### Running MMLU
 
-Below is an example of running MMLU on a local model with minimal tasks:
+Below is an example of running MMLU with a local safetensors model directory:
 
 ```bash
-$ export INSTRUCTLAB_EVAL_MMLU_MIN_TASKS=true   # don't set this if you want to run full MMLU
 $ export ILAB_MODELS_DIR=$HOME/.local/share/instructlab/models
-$ ilab model evaluate --benchmark mmlu --model $ILAB_MODELS_DIR/instructlab/granite-7b-lab
+$ ilab model evaluate --benchmark mmlu --model $ILAB_MODELS_DIR/instructlab/granite-7b-test
 ...
 # KNOWLEDGE EVALUATION REPORT
 
-## MODEL
-/home/example-user/.local/share/instructlab/models/instructlab/granite-7b-lab
-
-### AVERAGE:
-0.45 (across 3)
-
-### SCORES:
-mmlu_abstract_algebra - 0.35
-mmlu_anatomy - 0.44
-mmlu_astronomy - 0.55
+## MODEL (SCORE)
+/home/user/.cache/instructlab/models/instructlab/granite-7b-test/ (0.52/1.0)
```

Comment:
I noticed you added trailing slash at the end of `granite-7b-test/` and before there were none. Is that how the output looks or was there some other reason?

Reply:
Good catch!  I had it on the input side when I ran it to it was shown on the output.  Fixed.

Reply:
I don't see any change, maybe you forgot to push your changes?

Your change just makes `/home/user/.cache/instructlab/models/instructlab/granite-7b-test/ (0.52/1.0)` into `/home/user/.cache/instructlab/models/instructlab/granite-7b-test (0.52/1.0)` right?

Reply:
Sorry, I pushed the wrong branch, should be fixed now.